app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
+  margin-bottom: $base-spacing * 4;
+
+  .pagination {
```

Comment:
@tysongach Without this style, there wouldn't the right amount of space between the pagination links and text indicating the total.